app/src/full/java/io/homeassistant/companion/android/sensors/GeocodeSensorManager.kt

```diff
@@ -92,7 +92,7 @@ class GeocodeSensorManager : SensorManager {
                 )
             }.orEmpty()
 
-            val prettyAddress = address?.getAddressLine(0) ?: "Unknown"
+            val prettyAddress = address?.getAddressLine(0) ?: context.getString(R.string.state_unknown)
```

Comment:
This variable is used for the state of the sensor, we shouldn't translate the state string that we send back. Those need to be in English

Reply:
Thanks, i didn't know that. Fixed that 👍